Gemfile

```diff
@@ -162,9 +163,8 @@ group :development do
   gem "web-console", ">= 3.3.0"
 
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
-  gem "spring"
+  gem "spring", "~> 4.0"
   gem "spring-commands-rspec", "~> 1.0"
-  gem "spring-watcher-listen", "~> 2.0.0"
```

Comment:
Had to remove this gem if we want to upgrade to `spring ~>4`. Any objects? Is anyone using it?